### DISCUSSION TITLE
Загрузка картинок через буфер обмена

### DISCUSSION THREAD

**Discussion by smplseven** [totumonline/totum-ru-issues-and-discussions#331] [Есть идея, а что если...]
У нас часто возникает необходимость загрузки картинок в поле "файл". 
Сейчас эта возможность  открыта только путём перетаскивания самой картинки из рабочего стола или же поиска картинки на пк.

Не планируется ли  это сделать через CTRL  C -> CTRL +V ?

**Comment by ttm-alexeypolunin** [comment]
Врядли если чесн, тк мы взяли готовый JS загрузчик где-то на просторах интернета. Докручивать его дорого, а профит для нас нулевой.

**Reply by smplseven** [reply]
Хорошо,спасибо за ответ

**Reply by ttm-alexeypolunin** [reply]
Скопировать файл из буфера обмена не выходит - в ФФ вообще не работает чтение из буфера, Хром получает только текстовые файлы с названием, как у чуваков из примера удалось получить картинки - загадка. Интерфейс сильно урезан, при первом обращении требует от пользователя согласия -  чтобы страницы не следили за пользователем и не перехватывали критичные данные типа номеров карт.

https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/read

Так что это будет возможно только когда будет десктоп-версия на электроне.  Версия на электроне невозможна без инвестиций. Инвестиции невозможны, пока мы в РФ. Короч лет через 5 если повезет :)
